src/helpers/observer.js

```diff
@@ -16,13 +16,23 @@ const TOLERANCE_PX = 10;
 const {scrollIfNeeded, resetScrolling} = makeScroller();
 let next;
 
+let mouseX = 0;
+let mouseY = 0;
+
+function onMouseMove(event) {
```

Comment:
pointerAction already listens on mousemove/touchmove and has the info. i wonder if it could be leveraged.